pkg/ddc/thin/transform_fuse_test.go

```diff
@@ -705,7 +706,7 @@ func TestThinEngine_transformFuseWithDuplicateOptionKey(t1 *testing.T) {
 			}},
 			// ConfigValue: "{\"/thin/fluid/test/thin-fuse\":\"a=b\"}",
 			// MountPath:   "/thin/fluid/test/thin-fuse",
-			ConfigValue:   "{\"mounts\":[{\"mountPoint\":\"abc\",\"options\":{\"a\":\"b\"}}],\"targetPath\":\"/thin/fluid/test/thin-fuse\"}",
+			ConfigValue:   "{\"mounts\":[{\"mountPoint\":\"abc\",\"options\":{\"a\":\"b\"}}],\"targetPath\":\"/thin/fluid/test/thin-fuse\",\"accessModes\":[\"ReadOnlyMany\"]}",
```

Comment:
Could you add a test with accessModes as ReadWriteMany?

Reply:
see above code line 490,  accessModes as ReadWriteMany.